--- python38-pyasn1-epel.spec	2023-08-01 18:14:19.579002187 +0200
+++ python39-pyasn1-epel.spec	2023-08-01 18:44:12.037903047 +0200
@@ -1,10 +1,10 @@
-%global python3_pkgversion 38
+%global python3_pkgversion 39
 %global module pyasn1
 %global modules_version 0.2.8
 
 Name:           python%{python3_pkgversion}-pyasn1-epel
 Version:        0.4.8
-Release:        1%{?dist}
+Release:        1%{?dist}.srce
 Summary:        ASN.1 tools for Python
 License:        BSD
 Source0:        https://github.com/etingof/pyasn1/archive/v%{version}.tar.gz
@@ -18,6 +18,7 @@ language.
 
 %package -n python%{python3_pkgversion}-pyasn1
 Summary:    ASN.1 tools for Python 3
+Provides:   python3.9dist(pyasn1)
 BuildRequires:  python%{python3_pkgversion}-devel
 BuildRequires:  python%{python3_pkgversion}-setuptools
 
@@ -27,20 +28,12 @@ language.
 
 %package -n python%{python3_pkgversion}-pyasn1-modules
 Summary:    Modules for pyasn1
+Provides:   python3.9dist(pyasn1-modules)
 Requires:   python%{python3_pkgversion}-pyasn1 >= 0.4.7, python%{python3_pkgversion}-pyasn1 < 0.6.0
 
 %description -n python%{python3_pkgversion}-pyasn1-modules
 ASN.1 types modules for python%{python3_pkgversion}-pyasn1.
 
-%package doc
-Summary:        Documentation for pyasn1
-BuildRequires:  make
-BuildRequires:  python3-sphinx
-
-%description doc
-%{summary}.
-
-
 %prep
 %setup -n %{module}-%{version} -q -b1
 
@@ -52,11 +45,6 @@ pushd ../pyasn1-modules-%{modules_versio
 %py3_build
 popd
 
-pushd docs
-PYTHONPATH=%{buildroot}%{python3_sitelib} make SPHINXBUILD=sphinx-build-3 html
-popd
-
-
 %install
 %py3_install
 
@@ -79,10 +67,6 @@ PYTHONPATH=%{buildroot}%{python3_sitelib
 %{python3_sitelib}/%{module}_modules/
 %{python3_sitelib}/%{module}_modules-%{modules_version}-*.egg-info/
 
-%files doc
-%license LICENSE.rst
-%doc docs/build/html/*
-
 %changelog
 * Mon Oct 24 2022 Orion Poplawski <orion@nwra.com> - 0.4.8-1
 - Python 3.8 package for EPEL
